Client/Frontend/Library/BookmarksPanel.swift

```diff
@@ -207,7 +207,11 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel {
             _ = profile.places.deleteBookmarkNode(guid: bookmarkNode.guid)
 
             tableView.beginUpdates()
-            bookmarkNodes.remove(at: indexPath.row)
+            if indexPath.section == BookmarksSection.bookmarks.rawValue {
```

Comment:
extremely minor nit: I'd reverse this `if` to explicitly check for `BookmarksSection.recent` as the special case

```
if indexPath.section == BookmarksSection.recent.rawValue {
    recentBookmarks.remove(at: indexPath.row)
} else {
    bookmarkNodes.remove(at: indexPath.row)
}
```